packages/commonwealth/server/migrations/20240521002131-group-permissions.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    return queryInterface.createTable(
+      'GroupPermissions',
+      {
+        group_id: {
+          type: Sequelize.INTEGER,
+          allowNull: true,
```

Comment:
If `group_id` is the PK, then should `allowNull` be false?

Reply:
Good catch! Fixed